crates/starknet-devnet-types/src/rpc/transactions/l1_handler_transaction.rs

```diff
@@ -26,13 +27,22 @@ const PREFIX_L1_HANDLER: FieldElement = FieldElement::from_mont([
     157895833347907735,
 ]);
 
-impl L1HandlerTransaction {
-    /// Instantiates a new `L1HandlerTransaction`.
-    pub fn with_hash(mut self, chain_id: Felt) -> Self {
-        self.transaction_hash = self.compute_hash(chain_id);
-        self
-    }
+#[derive(Debug, Clone, Default, Eq, PartialEq, Deserialize, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct L1HandlerTransaction {
```

Comment:
Just wondering: what is the motivation for moving this struct?

Reply:
to keep the implementation and definition in 1 file. It was hard for me to look at what I am removing